src/errors/context-error.ts

```diff
@@ -1,11 +1,11 @@
 const CONTEXT_NOT_FOUND = (contextName: string) => `Context "${contextName}" does not exists.`;
 const MISSING_CURRENT_CONTEXT = 'No context is set as current, please set a current context.';
 export const NO_CONTEXTS_SAVED = `Specify what AsyncAPI file to be used.
-These are your options to specify in the CLI what AsyncAPI file should be used:
- - You can provide a path to the AsyncAPI file: asyncapi <command> path/to/file/asyncapi
- - You can also pass a saved context that points to your AsyncAPI file: asyncapi <command> context-name
- - In case you did not specify a context that you want to use, the CLI checks if there is a default context and uses it. To set default context run: asyncapi context use mycontext
- - In case you did not provide any reference to AsyncAPI file and there is no default context, the CLI detects if in your current working directory you have files like asyncapi.json, asyncapi.yaml, asyncapi.yml. Just rename your file accordingly.
+No context has been set yet, 
+To use a context as current you need to create one, follow these steps - 
+ - Create a new context: asyncapi config context add <context-name> <file-path | URL>
+ - Set this context as current: asyncapi config context use <context-name>
+ - Try asyncapi --help to learn more about commands.
```

Comment:
@pranay202 sorry for pointing you in the wrong direction, I just ran the command in CLI to understand where this error is getting generated, and changing this error message wouldn't be possible because somehow CLI is printing this message when there is no context created and when the input is incorrect as well. I think we need to rework the error handling right now it is a mess. 

can you just write this 

```suggestion
These are your options to specify in the CLI what AsyncAPI file should be used:
 - You can provide a path to the AsyncAPI file: asyncapi <command> path/to/file/asyncapi
 - You can also pass a saved context that points to your AsyncAPI file: asyncapi <command> context-name
 - In case you did not specify a context that you want to use, the CLI checks if there is a default context and uses it. To set default context run: asyncapi config context use mycontext
 - In case you did not provide any reference to AsyncAPI file and there is no default context, the CLI detects if in your current working directory you have files like asyncapi.json, asyncapi.yaml, asyncapi.yml. Just rename your file accordingly.
```

Just changing the `asyncapi context use..` to `asyncapi config context use..` 

Also after this issue would you like to work on remodeling the error handing, I would love to help you with it?

Reply:
Sure. No worries, thank you guiding me!